app/models/knight.rb

```diff
@@ -1,6 +1,15 @@
 class Knight < Piece
 
-	def valid_move?
+	def valid_move?(x, y)
+		return false if !on_board?(x, y)
+
+		if ((x - x_coordinate).abs == 2 && (y - y_coordinate).abs == 1) ||

```

Comment:
would be great to abstract out the checks to private methods and use them here.. like you've done on the queen piece